content/blog/2016/2016-05-18-external-workspace-manager-plugin.adoc

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "External Workspace Manager Plugin"
+tags:
+- jenkins
+- pipeline
+- plugins
+author: alexsomai
+---
+
+Currently, Jenkins’ build workspace may become very large in size due to the fact that some compilers
+generate very large volumes of data. The existing plugins that share the workspace across builds are able
+to do this by copying the files from one workspace to another, process which is inefficient.
+A solution is to have a Jenkins plugin that is able to manage and reuse the same workspace between multiple
+builds.
+
+As part of the link:https://summerofcode.withgoogle.com/[Google Summer of Code] 2016 I will be working on
+the External Workspace Manager plugin. This plugin aims to provide an external workspace management system.
+It should facilitate workspace share and reuse across multiple Jenkins jobs. It should eliminate the need
+to copy, archive or move files. The plugin will be written for
+link:https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Plugin[Pipeline] jobs.
+
```

Comment:
Here I would add several examples how is it going to work: short description of use-cases and one or two Pipeline snippets